pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -112,6 +116,14 @@ func (self *HandlerCreator) menuPrompt(prompt *config.CustomCommandPrompt, wrapp
 	return self.c.Menu(types.CreateMenuOptions{Title: prompt.Title, Items: menuItems})
 }
 
+func (self *HandlerCreator) confirmPrompt(prompt *config.CustomCommandPrompt, handleConfirm func() error) error {
+	return self.c.Confirm(types.ConfirmOpts{
+		Title:         self.c.Tr.ConfirmPromptTitle,
```

Comment:
I'd use `prompt.Title` here, just given that the user may want to specify it themselves. If it's left empty there's no issue with a blank title

Reply:
Got it. Do you think I should remove `ConfirmPromptTitle` and its associated translations then? Or that should be the default title?

Reply:
I'd just remove it. The less code, the better!

Reply:
Cool, sounds good to me! Just removed the extra code: f79c2d53910448c3e2248cd70d6a201dd02481ef